container/lxd/export_test.go

```diff
@@ -53,6 +53,10 @@ func PatchGetSnapManager(patcher patcher, mgr SnapManager) {
 	patcher.PatchValue(&getSnapManager, func() SnapManager { return mgr })
 }
 
+func PatchMaybeRemovePortFromOvsBridge(patcher patcher, fn func(string) error) {
```

Comment:
In my view, the only solution to this is to move these (unit) tests into the same package as the SUT so the PatchXYZ cruft and the export_test bits are not required. External packages should be able to mock the exported calls instead of messing with the package internals.